src/pybind/mgr/dashboard/tox.ini

```diff
@@ -153,7 +153,7 @@ commands =
 
 [testenv:check]
 commands =
-    python ci/check_grafana_dashboards.py frontend/src/app ../../../../monitoring/grafana/dashboards
+    python ci/check_grafana_dashboards.py frontend/src/app ../../../../monitoring/ceph-mixin/dashboards_out
```

Comment:
For non cephadm users, there's a way for them to publish these Grafana JSONs to their Grafana instance:

https://github.com/ceph/ceph/blob/e911f41672fb9bafdb4b4b053998fb275c81f70d/src/pybind/mgr/dashboard/grafana.py#L103-L110

So it'd be nice to update that location too.